update_prevalence.py

```diff
@@ -730,6 +742,13 @@ def main() -> None:
                         line.Seven_day_smoothed_daily_change * 7
                     )
 
+        # Romanian county (judet) data. Treat as states internally.
```

Comment:
I think you want Romania to be the State and the judet to be the County? Unless you expect sub-county data in the future...

(This is similar to how the UK works)

Reply:
https://en.wikipedia.org/wiki/Administrative_divisions_of_Romania
https://en.wikipedia.org/wiki/Jude%C8%9B
A județ (Romanian county) is the largest administrative division after "entire country", which is why I classified it as a "state" in update_prevalence.py.